### PR TITLE
.org with .com replaced

### PR DESCRIPTION
**Pull request by esgarov**
Only strings changed. doc.owncloud.org replaced with doc.owncloud.com

### CODE REVIEW

File: changelog/CHANGELOG-html.tmpl [hunk 1]
```diff
@@ -1224,7 +1224,7 @@
 <h4>External storage</h4>
 <ul>
     <li>&quot;Local&quot; storage type can now be disabled by sysadmin in config.php <a href="https://github.com/owncloud/core/issues/26653" target="_blank" rel="noopener">#26653</a></li>
-    <li>External storage backends must use <a href="https://doc.owncloud.org/server/10.0/developer_manual/app/extstorage.html" target="_blank" rel="noopener">core external storage API</a> to work without &quot;files_external&quot; <a href="https://github.com/owncloud/core/issues/18160" target="_blank" rel="noopener">#18160</a></li>
+    <li>External storage backends must use <a href="https://doc.owncloud.com/server/10.0/developer_manual/app/extstorage.html" target="_blank" rel="noopener">core external storage API</a> to work without &quot;files_external&quot; <a href="https://github.com/owncloud/core/issues/18160" target="_blank" rel="noopener">#18160</a></li>
```
Comment (mmattel) [inline changelog/CHANGELOG-html.tmpl:1227]:
```suggestion
    <li>External storage backends must use <a href="https://doc.owncloud.com/server/next/developer_manual/app/advanced/extstorage.html" target="_blank" rel="noopener">core external storage API</a> to work without &quot;files_external&quot; <a href="https://github.com/owncloud/core/issues/18160" target="_blank" rel="noopener">#18160</a></li>
```

File: CHANGELOG.html [hunk 2]
```diff
@@ -7424,7 +7424,7 @@ <h4>User management</h4>
 <h4>External storage</h4>
 <ul>
     <li>&quot;Local&quot; storage type can now be disabled by sysadmin in config.php <a href="https://github.com/owncloud/core/issues/26653" target="_blank" rel="noopener">#26653</a></li>
-    <li>External storage backends must use <a href="https://doc.owncloud.org/server/10.0/developer_manual/app/extstorage.html" target="_blank" rel="noopener">core external storage API</a> to work without &quot;files_external&quot; <a href="https://github.com/owncloud/core/issues/18160" target="_blank" rel="noopener">#18160</a></li>
+    <li>External storage backends must use <a href="https://doc.owncloud.com/server/10.0/developer_manual/app/extstorage.html" target="_blank" rel="noopener">core external storage API</a> to work without &quot;files_external&quot; <a href="https://github.com/owncloud/core/issues/18160" target="_blank" rel="noopener">#18160</a></li>
```
Comment (mmattel) [inline CHANGELOG.html:7427]:
```suggestion
    <li>External storage backends must use <a href="https://doc.owncloud.com/server/next/developer_manual/app/advanced/extstorage.html" target="_blank" rel="noopener">core external storage API</a> to work without &quot;files_external&quot; <a href="https://github.com/owncloud/core/issues/18160" target="_blank" rel="noopener">#18160</a></li>
```

Comment (mmattel) [inline CHANGELOG.html:7427]:
```suggestion
    <li>External storage backends must use <a href="https://doc.owncloud.com/server/next/developer_manual/app/advanced/extstorage.html" target="_blank" rel="noopener">core external storage API</a> to work without &quot;files_external&quot; <a href="https://github.com/owncloud/core/issues/18160" target="_blank" rel="noopener">#18160</a></li>
```

**Comment by mmattel** [comment]
Changed the URL to point to next and fixed the link so it is working

**Review by mmattel — DISMISSED** [review]

**Review by jnweiger — APPROVED** [review]
I believe, we correctly added @mmattel suggestions now.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**